platforms/ide/problems-api/src/main/java/org/gradle/api/problems/internal/InternalProblemSpec.java

```diff
@@ -48,10 +48,13 @@ public interface InternalProblemSpec extends ProblemSpec {
     InternalProblemSpec documentedAt(DocLink doc);
 
     @Override
-    InternalProblemSpec label(String label);
```

Comment:
Should we keep it as a deprecated method? It would probably make plugin developer's life's a bit easier. At the same time, they can depend on the Gradle 8.8 API and use reflection for earlier versions.

Reply:
I wouldn't keep things around as long as we are incubating.